app/views/admin/people/_form.html.erb

```diff
@@ -106,6 +106,6 @@
        }
     } %>
 
-    <%= link_to "Cancel", admin_people_path, class: "govuk-link" %>
+    <%= link_to "Cancel", :back, class: "govuk-link govuk-link--no-visited-state" %>
```

Comment:
the reason i'd generally advise that we don't use the referrer to set the backlink is that it doesn't behave nicely when the page reloads after an unsuccessful save 

https://github.com/alphagov/whitehall/assets/42515961/cd6a0073-1e2f-402b-a1d2-54bcbabb5cac

I can completely see why we would want to implement something that can set the cancel link back to the people page since we link there from the summary card, but i think unfortunately we will have to do it in a more hands on way which we should handle in another card.

Previously we've done stuff like pass a query string like `return_to=path_to_return_to` then handle routing in the controller via before actions 

Something like

```
before_action : set_cancel_link

def set_cancel_link
 @cancel_link =  if params[:return_to] == "people"
                               admin_organisation_people_path(@organisation)
                            else
                              admin_people_path
                            end 
```

def worth doing but also a lot of work do i wouldn't worry about it for now

Reply:
Cool, thanks @davidgisbey. I did wonder if the back link was a bit to simple to be true. We've implemented the conditional cancel link on params.